Doc/howto/logging.rst

```diff
@@ -296,9 +296,9 @@ which should print something like this:
 
    2010-12-12 11:41:42,612 is when this event was logged.
 
-The default format for date/time display (shown above) is ISO8601. If you need
-more control over the formatting of the date/time, provide a *datefmt*
-argument to ``basicConfig``, as in this example::
+The default format for date/time display (shown above) is like ISO8601 or
```

Comment:
This is unnecessary. It is not "like" ISO8601, it *is* ISO 8601.

Reply:
For now, I will sync the changes to be in line with the other branches, but won't close the issue. I think it's best just to state what the format is, make a reference to RFC 3339, and leave it at that. I will address this in a further update.